src/stacks/index.ts

```diff
@@ -7,3 +7,4 @@ export * from './nestedApiGateway';
 export * from './nestedRDS';
 export * from './nestedEKS';
 export * from './monitoring';
+export * from './nestedAppsync';
```

Comment:
- missing newline :smile: 
- I would keep the same case for all `*AppSync` wordings for consistency, so: `nestedAppSync` here - and there are other places in the PR in which `sync` is lowercase while `App` is uppercase which IMO could get updated